modules/Sources/Features/SendFlow/SendFlowStore.swift

```diff
@@ -79,7 +79,8 @@ public struct SendFlowReducer: Reducer {
         public var isValidForm: Bool {
             @Dependency(\.zcashSDKEnvironment) var zcashSDKEnvironment
 
-            return transactionAmountInputState.amount.data > zcashSDKEnvironment.network.constants.defaultFee().amount
+            return transactionAmountInputState.amount.data + zcashSDKEnvironment.network.constants.defaultFee().amount
```

Comment:
This is a very different check, and I think maybe not the one you want. It says that the input amount plus the fee must be a valid Zcash amount; is the `maxValue` here derived from the user's balance? It also can have false positives if the ZIP 317 fee would cause the actual balance of the transaction to exceed available funds. Ultimately what is probably needed here is a way to obtain the maximum spendable value as part of the wallet state, and check against that.

Reply:
I actually reverted the code because any change here is put of the scope of this ticket. If any change is needed we will create a follow up ticket.